lib/http.js

```diff
@@ -264,7 +269,6 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     }));
 
   // insert an authorization
-  app.options(routes.authorizations, cors());
```

Comment:
This was removed but `cors()` is still below?

Reply:
Yeah, I should have just kept it at the top. I was originally moving the `app.options` stuff to be next to all the `app.get` calls ... until I realized we don't have an `app.get` for everything, so I moved things back to the first instance of a path being used and missed this one. Functionality doesn't change, it's just inconsistent now.